packages/ui/src/theme/css/component/table.scss

```
@@ -0,0 +1,121 @@
+.amplify-table {
+  /**
+   * Default Table primitive stylings
+   */
+  border-collapse: collapse;
+  width: 100%;
+  display: table;
+  box-sizing: border-box;
+  text-indent: initial;
+
+  .amplify-table__head {
+    display: table-header-group;
+    vertical-align: middle;
+    border-color: inherit;
+  }
+
+  .amplify-table__body {
+    display: table-row-group;
+    vertical-align: middle;
+    border-color: inherit;
+  }
+
+  .amplify-table__foot {
+    display: table-footer-group;
+    vertical-align: middle;
+    border-color: inherit;
+  }
+
+  .amplify-table__row {
+    display: table-row;
+    vertical-align: inherit;
+    border-color: inherit;
+  }
+
+  .amplify-table__th,
+  .amplify-table__td {
+    display: table-cell;
+    vertical-align: inherit;
+    border-color: var(--amplify-components-table-border-color);
+    border-style: var(--amplify-components-table-border-style);
+    border-width: var(--amplify-components-table-border-width);
+    padding: var(--amplify-components-table-padding);
+    color: var(--amplify-components-table-color);
+    font-size: var(--amplify-components-table-font-size-default);
+  }
+
+  .amplify-table__th {
+    font-weight: var(--amplify-components-table-font-weight-th);
+  }
+
+  .amplify-table__td {
+    font-weight: var(--amplify-components-table-font-weight-td);
+  }
+
+  .amplify-table caption {
```

We should use the BEM element classname for consistency.
```suggestion
  .amplify-table__caption {
```

---

Added!